app/services/ToolManagerPlugin.scala

```diff
@@ -310,7 +310,7 @@ class ToolManagerPlugin(application: Application) extends Plugin {
       val apipath = play.Play.application().configuration().getString("toolmanagerURI") + "/instances/" + instance.toolPath
 
       val statusRequest: Future[Response] = url(apipath).put(Json.obj(
-        "dataset" -> (dsURL.replace("/datasets", "/api/datasets")+"/download"),
+        "dataset" -> (dsURL.replace("/datasets", "/api/datasets") + ß"/download"),
```

Comment:
not sure the ß is supposed to be there.

Reply:
This is corrected, sorry about that. Probably was an accidental keystroke.